bedrock/base/templates/404.html

```diff
@@ -2,47 +2,66 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% set_lang_files "mozorg/404" %}
-
-{% extends "base-pebbles.html" %}
+{% set_lang_files "mozorg/404-update-2020" %}
+{% extends "base-error.html" %}
 
 {% block gtm_page_id %}data-gtm-page-id="404"{% endblock %}
 
 {% block page_title %}{{ _('404: Page Not Found') }}{% endblock %}
 
-{# 404 pages must not include canonical and hreflang x-default: https://github.com/mozilla/bedrock/issues/5895 #}
-{% block canonical_urls %}{% endblock %}
-
 {% block page_css %}
   {{ css_bundle('page_not_found') }}
 {% endblock %}
 
+{# 404 pages must not include canonical and hreflang x-default: https://github.com/mozilla/bedrock/issues/5895 #}
+{% block canonical_urls %}{% endblock %}
+
 {% block content %}
-<main>
-  <div class="content">
-    <header>
-      <h1><span>{{ _('Whoops!') }}</span></h1>
-      <p>{{ _('Did you make a left at that last URL instead of a right? No problem. Here are some tips to get you back on your way:') }}</p>
-    </header>
-    <div class="error-content">
-      <ul class="prose">
-        <li>{{ _('If you typed in the address, check your spelling. Could just be a typo.') }}</li>
+  <main role="main">
+    <section class="mzp-l-content message-section">
+      <h1 class="error_header">{{ _('Sorry, we can’t find that page') }}</h2>
+      <p>{{_('We’re all about a healthy internet but sometimes broken URLs happen.') }}</p>
+      <div class="hide-back" id="go-back">
+        <a class="back-button" onclick="goBack()">{{ _("Go Back") }}</a>
```

Comment:
Generally we should try and avoid creating global functions to handle click events. Suggest moving this `onClick` handler to the script you created and using `addEventListener()` to handle the event?

The link should probably also have an `href='#'` value.

Reply:
```suggestion
        <a class="back-button" onclick="goBack()">{{ _('Go Back') }}</a>
```